package.json

```diff
@@ -36,19 +36,27 @@
     },
     "devDependencies": {
         "@playwright/test": "^1.39.0",
-        "ajv": "^8.11.0",
+        "@types/node": "^20.8.10",
+        "@zip.js/zip.js": "^2.7.30",
+        "ajv": "^8.12.0",
         "browserify": "^17.0.0",
         "css": "^3.0.0",
+        "dexie": "^3.2.4",
+        "dexie-export-import": "^4.0.7",
+        "esbuild": "^0.19.5",
         "eslint": "^8.52.0",
         "eslint-plugin-header": "^3.1.1",
         "eslint-plugin-jsdoc": "^46.8.2",
         "eslint-plugin-jsonc": "^2.10.0",
         "eslint-plugin-no-unsanitized": "^4.0.1",
         "fake-indexeddb": "^5.0.1",
+        "handlebars": "^4.7.8",
         "html-validate": "^8.7.0",
         "jsdom": "^22.1.0",
+        "jszip": "^3.10.1",
```

Comment:
the tests haven't been moved from jszip to zip.js yet, but i think this dependency isn't currently being used

Reply:
I think zip.js might be hard to use in node but I will take another shot in another PR at some point.